llvm/test/CodeGen/X86/isel-x87.ll

```diff
@@ -0,0 +1,225 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=i686-- -mattr=+x87,-sse,-sse2 -global-isel | FileCheck %s --check-prefixes=CHECK-32,GISEL_X86
```

Comment:
Move under the GlobalISel test subdirectory? I also wouldn't add in the fast-isel lines